src/components/pages/LoginPage.js

```diff
@@ -1,5 +1,45 @@
+import styled from "styled-components";
+import "font-awesome/css/font-awesome.min.css";
```

Comment:
It is necessary if we want to have a FA icon inside a placeholder. Moreover we can use this package with other icons too. I asked TL about it i waiting for decision.

Reply:
And Yes i followed FA docu and many other sources :)

Reply:
We already have some packages ([https://github.com/CC2021-WBL/GOOD-BOI-Application/blob/development/package.json]) from fontawesome (fortawesome) installed on development. Is there a way to use it instead of adding this? It seems its version 4 also, perhaps there is a way to use the ones we already have? It seems a bit overkill to install the full package for these 2 icons.

Reply:
I'm sure that other icons used with this package we have, are possible with my package too. It is an overkill to install 2 packages of course but maybe we can just change the package and save some time.